core/domain/activity_jobs_one_off.py

```diff
@@ -55,3 +56,39 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         pass
+
+
+class ReplaceAdminIdOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that replaces the usage of 'Admin' in ExplorationCommitLogEntryModel
+    and ExplorationRightsSnapshotMetadataModel.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationRightsSnapshotMetadataModel,
+                exp_models.ExplorationCommitLogEntryModel]
+
+    @staticmethod
+    def map(model):
+        if isinstance(model, exp_models.ExplorationRightsSnapshotMetadataModel):
+            if model.committer_id == 'Admin':
+                model.committer_id = feconf.SYSTEM_COMMITTER_ID
+                model.put(update_last_updated_time=False)
+                yield ('SUCCESS-DELETED-SNAPSHOT', model.id)
```

Comment:
"deleted" seems a strange way to phrase this, since we aren't deleting anything?

Ditto below.

Reply:
Done.